dashboard/config/environments/test.rb

```diff
@@ -21,7 +21,13 @@
   config.public_file_server.enabled = true
   config.public_file_server.headers = { 'Cache-Control' => "public, max-age=3600, s-maxage=1800" }
 
-  # test environment should use precompiled digested assets like production,
+  # Whether or not to display pretty apps (formerly called blockly).
+  config.pretty_apps = false
+
+  # Whether or not to display pretty shared js assets
+  config.pretty_sharedjs = true
```

Comment:
Just confirming, per comment below clarifying `ENV['CI']`, this would change `pretty_sharedjs` to newly be true on the main test machine.